app/assets/javascripts/alchemy/alchemy.sitemap.js.coffee

```diff
@@ -8,7 +8,7 @@ Alchemy.Sitemap =
     @search_field = $("#search_field")
     @filter_field_clear = $('.js_filter_field_clear')
     @display = $('#page_filter_result')
-    @sitemap_wrapper = $('#sitemap-wrapper .sitemap-load-notice')
+    @sitemap_wrapper = $('#sitemap-wrapper')
```

Comment:
I see you're using `.replaceWith` instead of `.html` that indeed replaces the element. Why?